packages/aesirx-dma-app/src/containers/ChannelsPage/ChannelsList/ChannelType.jsx

```diff
@@ -41,24 +41,26 @@ const ChannelType = observer(({ channelTypeIndex, channelCategory }) => {
     <div className="accordion mt-4" id={`accordionChannelType${channelTypeIndex}`}>
       {list.map((channelType, index) => (
         <div
-          className="accordion-item mt-3 bg-blue-5 rounded-2 border-0"
+          className="accordion-item mt-24 bg-blue-5 rounded-2 border-0"
           key={Math.random(40, 200)}
         >
           <h3 className="accordion-header" id={`flush-heading${index}`}>
             <div
-              className={`p-3 d-flex align-items-center`}
+              className={`d-flex align-items-center accordion-item-custom`}
```

Comment:
`accordion-item-custom` just add padding so you can use `py` or `px` class instead of a custom class